Sources/AsyncHTTPClient/ConnectionPool.swift

```diff
@@ -139,12 +139,16 @@ final class ConnectionPool {
             self.port = request.port
             self.host = request.host
             self.unixPath = request.socketPath
+            if let tls = request.tlsConfiguration {
```

Comment:
@artemredkin should we add a test case that targets the pool directly here? Ie no actual connections?